pkgs/by-name/ud/udev-block-notify/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  multimarkdown,
+  coreutils,
+  libnotify,
+  udev,
+  systemdLibs,
+  glib,
+  pkg-config,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "udev-block-notify";
+  version = "0.7.11";
+
+  src = fetchFromGitHub {
+    owner = "eworm-de";
+    repo = "udev-block-notify";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-A0uhfb2mEAAJgxRkv+MWTk/9oFiz3r7deAlu1Kpk+CI=";
+  };
+
+  nativeBuildInputs = [
+    multimarkdown
+    coreutils
```

Comment:
```suggestion
```

Not needed

Reply:
I wasn't sure if install was always available without it, thanks